client/src/components/ProfileEdit.jsx

```diff
@@ -9,7 +9,12 @@ class ProfileEdit extends Component {
     state = {
         user_edit: {},
         error_nick: "",
-        error_repeated: ""
+        error_repeated: "",
+        error_first: "",
+        error_last: "",
+        error_trigger_nick: false,
+        error_trigger_first: false,
+        error_trigger_last: false
```

Comment:
I don't want to know why did you do this and how it works. But it seems really strange.